defaults/main.yml

```diff
@@ -34,6 +34,7 @@ nginx_worker_rlimit_nofile: 1024
 nginx_log_dir: "/var/log/nginx"
 nginx_error_log_level: "error"
 
+nginx_extra_params: ""
```

Comment:
nginx_extra_params
Maybe we can make this more explicit like "nginx_extra_root_params" or "nginx_extra_top_params"